templates/default/mesos_slave.yaml.erb

```diff
@@ -0,0 +1,13 @@
+# Generated by Chef, local modifications will be overwritten
```

Comment:
A custom template was used to allow the defaults to be used without having to define extra attributes